src/tactic/ui/container/tab_wdg.py

```diff
@@ -38,6 +38,11 @@ class TabWdg(BaseRefreshWdg):
              'values': 'true|false',
             'category': 'Display'
         },
+        'save_state': {
+            'description': 'key which is used to save state',
```

Comment:
Yes, any tab can specify a save state: you just have to specify a unique key to store this under.